### PR TITLE
Add SDK version warning to AI tutorial

### PR DESCRIPTION
**Pull request by mikkelhegn**
Adding a small disclaimer that code samples in the article are for older SDK versions.

### CODE REVIEW

**Comment by mikkelhegn** [comment]
Inspired by: https://discord.com/channels/926888690310053918/1148522244277014568/1313066570079146024

File: content/spin/v3/ai-sentiment-analysis-api-tutorial.md [hunk 1]
```diff
@@ -42,6 +42,8 @@ In this tutorial we will:
 
 ## Tutorial Prerequisites
 
+> Please note that the code sample in this article is not using the latest version of the Spin SDK, but works with the latest version of Spin (3.0.0). Please refer to the code in this [GitHub repository for the working samples and corresponding SDK version](https://github.com/fermyon/ai-examples/tree/main).
```
Comment (karthik2804) [inline content/spin/v3/ai-sentiment-analysis-api-tutorial.md:45]: I do not think we made any breaking changes regarding the SDK, and things should work, which confuses me.

Comment (mikkelhegn) [reply content/spin/v3/ai-sentiment-analysis-api-tutorial.md:45]: I tried with the Rust sample and I had to change it to make it work.

This is the diff I ended up with to make the Rust sample work:
`5c5
<     http::{IntoResponse, Params, Request, Response, Router},
---
>     http::{IntoResponse, Json, Params, Request, Response, Router},
56c56,59
< fn perform_sentiment_analysis(req: Request, _params: Params) -> Result<impl IntoResponse> {
---
> fn perform_sentiment_analysis(
>     req: http::Request<Json<SentimentAnalysisRequest>>,
>     _params: Params,
> ) -> Result<impl IntoResponse> {
58,60c61
<     let sentiment_analysis_request: SentimentAnalysisRequest =
<         serde_json::from_slice(req.body()).expect("Failed to deserialize request body");
<     let sentence = sentiment_analysis_request.sentence.trim();
---
>     let sentence = req.body().sentence.trim();`

Comment (mikkelhegn) [reply content/spin/v3/ai-sentiment-analysis-api-tutorial.md:45]: Error with the orginal code 
```
├╴E failed to resolve: use of undeclared crate or module `http`
 │   use of undeclared crate or module `http` rustc (E0433) [57, 10]
`
```

Never templates don't have 'http' as a dependency - can that be it?

Comment (mikkelhegn) [reply content/spin/v3/ai-sentiment-analysis-api-tutorial.md:45]: The sample in the repo, which works without any changes use a fairly old SDK `1.5.0`. I haven't checked if the other languages work based on new SDK version and "old" code sample.

**Review by itowlson — CHANGES_REQUESTED** [review]
Could we not update the code rather than telling people to go back to an old SDK?  Leading a tutorial with "this no longer works" bit feels kinda awkward.

**Comment by mikkelhegn** [comment]
Yes we could, but all the samples in the repo that is referred in the article would probably need updating, unless we're ok having the article and repo be out of sync: https://github.com/fermyon/ai-examples/tree/main

**Comment by mikkelhegn** [comment]
I've changed strategies and is now updating article and samples. Will close this and open a new PR.

**Pull request closed without merging**